apps/web/composables/usePayPal/usePayPal.ts

```diff
@@ -12,7 +11,21 @@ import type {
   CreateCreditCardTransaction,
   CaptureOrder,
   LoadConfig,
+  GetLocale,
 } from './types';
+import { paypalGetters } from '@plentymarkets/shop-sdk';
+
+const getLocaleForPayPal: GetLocale = (locale: string) => {
+  // eslint-disable-next-line sonarjs/no-small-switch
+  switch (locale) {
+    case 'de': {
+      return 'de_DE';
+    }
+    default: {
+      return 'en_US';
+    }
+  }
```

Comment:
Do you think it might be a good idea to put the paypalLocale in `i18n.config.ts` file ? If we add a new language tomorrow we would have to change in multiple places if we leave this here.

Reply:
But there are only number_formats in it, so Maxi and I decided to move it in the usePayPal